### PR TITLE
support laravel 9

### PR DESCRIPTION
**Pull request by thekonz**
I tried to update a project to laravel 9, but this package doesn't support it yet.
Tests ran without error. I could add some github actions later, if you want.

### CODE REVIEW

**Comment by thekonz** [comment]
ok, acions seem to be working well: https://github.com/thekonz/laravel-accept-language-middleware/actions/runs/1985905373

**Comment by orkhanahmadov** [comment]
Hi @thekonz 👋 
Thank you for your contribution with Laravel 9 support and Github action.
I'm thinking instead of carrying older PHP and Laravel versions maybe we just release a new major version with only `PHP:^8.0` and `Laravel:^9.0` support. Can you please update your PR to have this?

File: composer.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
-        "illuminate/support": "^6.0|^7.0|^8.0"
+        "php": "^8.0",
+        "illuminate/contracts": "^9.0",
+        "illuminate/http": "^9.0",
+        "illuminate/support": "^9.0"
     },
     "require-dev": {
-        "orchestra/testbench": "^4.0|^5.0|^6.0",
+        "orchestra/testbench": "^7.0",
         "phpunit/phpunit": "^8.0|^9.0"
```
Comment (orkhanahmadov) [inline composer.json]: PHPUnit 9 should be minimum as Laravel 9 does not support PHPUnit 8

**Review by orkhanahmadov — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (thekonz) [reply composer.json]: hm true, i just looked at orchestra/testbench

File: composer.json [hunk 2]
```diff
@@ -18,13 +18,13 @@
         }
     ],
     "require": {
-        "php": "^7.2|^8.0",
-        "illuminate/contracts": "^6.0|^7.0|^8.0",
-        "illuminate/http": "^6.0|^7.0|^8.0",
-        "illuminate/support": "^6.0|^7.0|^8.0"
+        "php": "^8.0",
```
Comment (orkhanahmadov) [inline composer.json]: Minimum PHP version should be `^8.0.2`

**Review by orkhanahmadov — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (thekonz) [reply composer.json]: yup, scrutinizer also doesn't like 8.0 :)

**Comment by orkhanahmadov** [comment]
Thank you! 🎉 

Tagged [`2.0.0`](https://github.com/orkhanahmadov/laravel-accept-language-middleware/releases/tag/2.0.0).

**Comment by thekonz** [comment]
thank you :rocket:

**Pull request merged**